packages/x-date-pickers/src/PickersShortcuts/PickersShortcuts.tsx

```diff
@@ -45,6 +45,16 @@ export interface PickersShortcutsProps<TValue> extends ExportedPickersShortcutPr
   isValid: (value: TValue) => boolean;
 }
 
+/**
+ * Demos:
+ *
+ * - [Custom slots and subcomponents](https://mui.com/x/react-date-pickers/custom-components/)
```

Comment:
WDYT about removing this page?
It does not list any demo of this component but links to the `Shortcuts` page. 🤔 
```suggestion
```